ambari-server/src/main/java/org/apache/ambari/server/agent/stomp/dto/MetadataCluster.java

```diff
@@ -87,7 +87,16 @@ public boolean updateServiceLevelParams(SortedMap<String, MetadataServiceInfo> u
         break;
       }
     }
+
+    for (String key : serviceLevelParams.keySet()) {
+      if (!update.containsKey(key) || !update.get(key).equals(update.get(key))) {
```

Comment:
There is a typo in this line: `update.get(key)` is compared to itself (`update.get(key)`) instead of `serviceLevelParams.get(key)`.

Further, I think this comparison can be removed: values for keys present in both maps are already compared in the first loop.

Reply:
Done